server/app/views/admin/questions/QuestionConfig.java

```diff
@@ -179,16 +179,22 @@ private static DivTag multiOptionQuestionField(
                 ImmutableSet.of(ValidationErrorMessage.create(MessageKey.MULTI_OPTION_VALIDATION)))
             .showFieldErrors(false)
             .getInputTag()
-            .withClasses("flex", "ml-2", "gap-x-3", ReferenceClasses.MULTI_OPTION_INPUT);
+            .withClasses(ReferenceClasses.MULTI_OPTION_INPUT);
     DivTag optionAdminName =
         FieldWithLabel.input()
             .setFieldName(isForNewOption ? "newOptionAdminNames[]" : "optionAdminNames[]")
-            .setLabelText("Admin Name")
+            .setLabelText("Admin ID")
```

Comment:
**thought:** I'm not sure what the right place is for this, but it may be worth mentioning that this field can't be changed in the future. Also that this is the text that will be exported in the CSV (but that feels like it can live in the docs)

Reply:
I added a tooltip icon but it made it look really busy, so I decided to wait until we had a full design to figure that out:
https://github.com/civiform/civiform/issues/5612

Reply:
Sounds good - as long as we make that clear with Arkansas in the meantime that makes sense.